package.json

```diff
@@ -4,13 +4,14 @@
   "version": "0.0.0",
   "main": "dist/index.html",
   "scripts": {
-    "start": "./node_modules/.bin/gulp",
-    "build": "./node_modules/.bin/gulp build",
-    "test": "./node_modules/.bin/gulp test",
-    "serve": "./node_modules/.bin/gulp serve",
-    "deploy": "./node_modules/.bin/gulp deploy"
+    "start": "gulp",
+    "build": "gulp build",
+    "test": "gulp test",
+    "serve": "gulp serve",
+    "deploy": "gulp deploy"
   },
   "dependencies": {
+    "gulp-cli": "github:gulpjs/gulp-cli#4.0",
```

Comment:
Absolutely, I think we should discuss it. I did this pragmatically to fix the build :) But I agree we should have a deeper discussion.